tRecorderApi/api/pipeline.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.hashers import make_password
+
+
+def user_created(strategy, details, user=None, *args, **kwargs):
+    print(user.username)
+
+    if user:
+        # password = make_password("some_password")
+
+        user.is_staff = True
```

Comment:
Does the guest account need to be staff?

Reply:
is_staff is here only to differentiate identicon users from social users. We can use different property name